doc/release-notes-31278.md

```diff
@@ -0,0 +1,7 @@
+Updated RPCs
```

Comment:
In "wallet,rpc: paytxfee and settxfee is deprecated and hidden from help" ee080ce689bfc8b74438a02e1c62bbe644893ffa

nit:
I think release notes is usually on it's own commit